src/cloud/docker/docker-multi-website.md

```diff
@@ -0,0 +1,39 @@
+---
+group: cloud-guide
+title: Set up multiple websites or stores
+functional_areas:
+  - Cloud
+  - Setup
+  - Configuration
+  - Website
+---
+
+Magento Cloud Docker supports multi-website by using different subdomains.
+
+## Add subdomains
+
+1. Ensure the `php.ini` file included and pints to correct `magento-vars.php` configration
+
+    ```ini
+    auto_prepend_file = /app/magento-vars.php
+    ```
+
+Magento Cloud Docker read the `php.ini` file if it is included in your project.
+
+1. Each subdomain must be added to `/etc/hosts` configuration file
+
+    ```
+    127.0.0.1 magento2.docker
+    127.0.0.1 second.magento2.docker
+    ```
+
+{:.bs-callout-info}
+You can make next changes while your Cloud Docker is running.
+
```

Comment:
```suggestion
    ```

    {:.bs-callout-info}
   You can make next changes while your Cloud Docker is running.

```